README.md

```diff
@@ -78,19 +78,33 @@ yourlogger.addHandler(airbrake.AirbrakeHandler())
 _by default, the `AirbrakeHandler` only handles logs level ERROR (40) and above_
 
 #### Additional Options
-More options are available to configure this library. Similar to setup, you can do this with environment variables
+More options are available to configure this library. 
+
+For example; to add more context to your errors, you can send the hostname. One way is by setting the HOSTNAME env var.
 ```
 export HOSTNAME=sassbox-101.prod.api
 ```
-Or when you instantiate the logger
+Or you can set it more explicitly when you instantiate the logger.
 ```python
 import airbrake
 
 logger = airbrake.getLogger(api_key=*****, project_id=12345, hostname='sassbox-101.prod.api')
 ```
 
 The available options are:
-- hostname (AIRBRAKE_HOSTNAME), default: `socket.gethostname()`
+- hostname, defaults to env var `HOSTNAME` or `socket.gethostname()`
+- environment, defaults to env var `AIRBRAKE_ENVIRONMENT` or `socket.gethostname()`
```

Comment:
Why does environment default to `socket.gethostname()`? This doesn't look correct to me.